### PR TITLE
Feature list registros

### PR DESCRIPTION
**Pull request by mikeyrayvon**
adds list of saved and submitted applications for admin & committee.  adds single application template for admin & committee

### CODE REVIEW

File: app/client/css/main.styl [hunk 1]
```diff
@@ -124,6 +125,21 @@ a,a:active,a:visited
...
 .preloader-wrapper.active
   margin: 0 $margin-basic
+  
+// Tables
+th.tablesorter-header
+  cursor: pointer
+  &:focus
+    outline: 0
+  
+.tablesorter-headerDesc
```
Comment (varffs) [inline app/client/css/main.styl:135]: @mikeyrayvon are these generated classes?

Comment (mikeyrayvon) [inline app/client/css/main.styl:135]: @varffs yea.  generated by the tablesorter package

Comment (mikeyrayvon) [inline app/client/css/main.styl:135]: code copied from material app

Comment (varffs) [inline app/client/css/main.styl:135]: yh thats cool then

File: app/client/templates/admin/submissions.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<template name="submissions">
+  <div class="container">
+    <table class="striped js-sortable" data-sortlist="[[2,1]]">
+      <thead>
+        <tr>
+            <th data-field="name">{{_ "application.name.label"}}</th>
+            <th data-field="project">{{_ "application.projectTitle.label"}}</th>
+            <th data-field="status">{{_ "application.status"}}</th>
```
Comment (varffs) [inline app/client/templates/admin/submissions.html:8]: Clicking on these headers seems to toggle them gray but do nothing else?

Comment (mikeyrayvon) [inline app/client/templates/admin/submissions.html:8]: do you have more than one submission?

**Pull request merged**